### ISSUE TITLE
[BUG] my account mmr is just broken and its surelly a bug

### ISSUE THREAD

**Issue by puchat1** [RiotGames/developer-relations#797]
**Bug Description**  
broken mmr

**Problem Description**  
getting +15 lp and losing -27 lp seems not normal 

**Expected Result**  
i expect you to fix this issue because its just uncomfortable to play with this type of mmr its basicly imposible to play with this kind of bug

**Actual Result**  
![obraz](https://github.com/RiotGames/developer-relations/assets/141188226/28a65315-ce91-4f0f-9917-67c9326378c1)

**Issue Comments**  
I'm almost 1v9ing every one of my games yet even when i won today 7 games and lost 5 i am on -100 points (doesn't seem so fair to me), my nick is mentälly stäble on EUNE and riot id-ambatublow, tag- #5558

**Comment by geozukunft** [comment]
This repo is for Issues related to the API of Riot Games and does not have anything to do with the game itself. 

If you truly believe it is a bug go to support.riotgames.com but trust me the system works properly

**Comment by puchat1** [comment]
I already did and i got absolutely no help, it has to be a bug , there is
no way i am getting +15 and -27lp ( my friend with the same winrate as me
gets +23 -24 lp which is reasonable )

śr., 2 sie 2023 o 00:28 Viktor geozukunft ***@***.***>
napisał(a):

**Comment by bangingheads** [comment]
That still does not make this the correct place to report this

**Comment by DoctressWasTaken** [comment]
Oh no, if it isn't the consequences of your own actions.

Ignoring that this is entirely the wrong place to ask this you brought this onto yourself, duoQing with another player that is 3 leagues below you. Of course you fuck over your MMR that way.
Good luck getting it back to normal 🙂

**Comment by puchat1** [comment]
dawg when i played soloq without him on normal emerald elo i got +15 so how
is this his fault

śr., 2 sie 2023 o 00:49 Robin ***@***.***> napisał(a):

**Comment by DoctressWasTaken** [comment]
Lemme rephrase it in simpler terms for you:

- You played duo with another player that is significantly lower ranked than you
- You won 57% of the games, barely higher than your average winrate
- Because you duoed with a player much lower the games you played were a lot easier, after all they were inbetween both of your MMR
- Based on your performance, the system now thinks that you are much worse than your current visible rank.
- The system is trying to put you back into your proper rank
- Thats why you are losing much more than you are winning

Play another 30-40 games solo and it might fix itself, maybe a bit more, who knows

> The losses will continue until ~~moral~~ mmr improves

**Comment by tisbells** [comment]
Hi there - We are only able to support issues related to our API. Please visit [Player Support](https://support.riotgames.com/hc/en-us) for game client issues.
